server/feeds/routes.ts

```diff
@@ -22,6 +23,81 @@ const log = debug("bobaserver:feeds:routes");
 
 const router = express.Router();
 
+/**
+ * @openapi
+ * /feeds/realms/{realm_id}:
+ *   get:
+ *     summary: Get latest activity on entire realm
+ *     operationId: getRealmActivity
+ *     tags:
+ *       - /feeds/
+ *     parameters:
+ *       - name: realm_id
+ *         in: path
+ *         description: The external id of the realm to fetch the activity of.
+ *         required: true
+ *         schema:
+ *           type: string
+ *       - name: cursor
+ *         in: query
+ *         description: The cursor to start feeding the activity of the board from.
+ *         schema:
+ *           type: string
+ *         allowEmptyValue: true
+ *     responses:
+ *        404:
+ *          description: The realm was not found.
+ *        200:
+ *         description: The realm's activity.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: "#/components/schemas/FeedActivity"
+ */
+router.get("/realms/:realm_id", ensureLoggedIn, async (req, res) => {
+  const { realm_id: realmExternalId } = req.params;
+  const { cursor } = req.query;
+  log(
+    `Fetching activity data for realm with slug ${realmExternalId} with cursor ${cursor}`
+  );
+
+  log(cursor);
+  const result = await getRealmActivityByExternalId({
+    realmExternalId,
+    firebaseId: req.currentUser?.uid || null,
+    cursor: (cursor as string) || null,
+  });
+  info(`Found activity for realm ${realmExternalId}:`, result);
+
+  if (result === false) {
```

Comment:
You don't need this because you haven't captured the exception in the query (as you shouldn't have). The server will automatically return 500 status if an exception occurs.